config/locales/user_mailer/en.yml

```diff
@@ -8,6 +8,29 @@ en:
         didn't request a password reset, you can ignore this message.
       link_text: Create your account
       subject: Your login.gov password reset request
+    account_reset_request:
+      cancel_link_text: click here
+      help: ''
+      intro: You are receiving this email because you requested to delete your login.gov
```

Comment:
This text is out of sync with the updates in the issue:

Delete your account
You’ve requested to delete your login.gov account.
Your request will be processed in 24 hours.
If you don’t want to delete your account, please cancel: [cancel link].